games-vue-client/src/components/ServerSelection.vue

```diff
@@ -47,12 +40,12 @@ export default {
   data() {
     return {
       serverOptions: serverOptions,
-      chosenServer: serverOptions[0].url
     };
   },
   computed: {
     serverInfo() {
-      return this.serverOptions.find(s => s.url === this.chosenServer);
+      const query = this.$route.query.server;
```

Comment:
What are your thoughts about this vs. `new URL(document.location).searchParams.get("server")` ?

If I understand correctly, using `this.$route.query.server` is `http://localhost:8080/#/login?server=localhost-development`
but using `new URL(document.location).searchParams` is `http://localhost:8080/?server=localhost-development#/login`

Which one to use?

Reply:
In some cases it might be very useful to use searchParams, for solutions that uses the hashtag.

Don't think it matters in this case since we should only collect the query once and not in every navigation. I hope that happens, realised I need to test more.